src/index.js

```diff
@@ -483,7 +481,7 @@ function createConfig(options, entry, format, writeMeta) {
 							!!writeMeta &&
 							options.css !== 'inline' &&
 							options.output.replace(
-								/(\.(umd|cjs|es|m))?\.(mjs|[tj]sx?)$/,
+								/(\.(umd|cjs|es|m))?\.([cm]js|[tj]sx?)$/,
```

Comment:
This was the only regex that needed to change, the others were functionally fine, but I updated them to match.